Geometry/DTGeometry/test/DTGeometryAnalyzer.cc

```diff
@@ -37,16 +40,26 @@ class DTGeometryAnalyzer : public edm::one::EDAnalyzer<> {
   const int dashedLineWidth_;
   const string dashedLine_;
   const string myName_;
+  bool tinyDifferences_;
```

Comment:
@cvuosalo - why not a `tolerance_` itself rather then the hardcoded numbers? I think, there are other analysers that define tolerance.